lib/msal-common/src/cache/entities/AccessTokenEntity.ts

```diff
@@ -4,164 +4,28 @@
  */
 
 import { CredentialEntity } from "./CredentialEntity";
-import { CredentialType, AuthenticationScheme } from "../../utils/Constants";
-import { TimeUtils } from "../../utils/TimeUtils";
-import { ICrypto } from "../../crypto/ICrypto";
-import { TokenClaims } from "../../account/TokenClaims";
-import {
-    createClientAuthError,
-    ClientAuthErrorCodes,
-} from "../../error/ClientAuthError";
-import { extractTokenClaims } from "../../account/AuthToken";
+import { AuthenticationScheme } from "../../utils/Constants";
 
 /**
- * ACCESS_TOKEN Credential Type
- *
- * Key:Value Schema:
- *
- * Key Example: uid.utid-login.microsoftonline.com-accesstoken-clientId-contoso.com-user.read
- *
- * Value Schema:
- * {
- *      homeAccountId: home account identifier for the auth scheme,
- *      environment: entity that issued the token, represented as a full host
- *      credentialType: Type of credential as a string, can be one of the following: RefreshToken, AccessToken, IdToken, Password, Cookie, Certificate, Other
- *      clientId: client ID of the application
- *      secret: Actual credential as a string
- *      familyId: Family ID identifier, usually only used for refresh tokens
- *      realm: Full tenant or organizational identifier that the account belongs to
- *      target: Permissions that are included in the token, or for refresh tokens, the resource identifier.
- *      cachedAt: Absolute device time when entry was created in the cache.
- *      expiresOn: Token expiry time, calculated based on current UTC time in seconds. Represented as a string.
- *      extendedExpiresOn: Additional extended expiry time until when token is valid in case of server-side outage. Represented as string in UTC seconds.
- *      keyId: used for POP and SSH tokenTypes
- *      tokenType: Type of the token issued. Usually "Bearer"
- * }
+ * Access token cache type
```

Comment:
Can we please retain the schema in the comments? Until we actually change the cache, we may need to keep the structure

Reply:
The schema is still documented, I only moved the comments to be above each field rather than all together in this big block. It allows Typedocs/Intellisense to accurately describe each field and is easier for maintenance

Reply:
I meant literally these lines, helps gauge structure of the entiry
 ```
 * Key:Value Schema:
 *
 * Key Example: uid.utid-login.microsoftonline.com-accesstoken-clientId-contoso.com-user.read
 ```

Reply:
Gotcha, I'll add this back to the key generation function. Does that work for you?

Reply:
Yes.